apps/tup-cms/src/taccsite_cms/settings_custom.py

```diff
@@ -192,6 +192,8 @@
 TACC_BLOG_CUSTOM_MEDIA_POST_CATEGORY = 'multimedia'
 TACC_BLOG_SHOW_ABSTRACT_TAG = 'external'
 
+TACC_EXCLUDED_POST_NOMIAL_LIST = ["Jr", "Sr", "I", "II", "III", "IV", "V", "VI", "VII", "VIII", "IX", "X"]
```

Comment:
**Cool!** Also, **name brainstorming**.[^1]

- `TACC_STAFF_PROFILE_…`
- `TACC_STAFF_…`
- ?

[^1]: **Justification:** I am recently trying to follow Django settings naming of successive prefixes, e.g. [Django CMS Blog uses `BLOG_`](https://djangocms-blog.readthedocs.io/en/stable/autodoc/settings.html#module-djangocms_blog.settings), [Django's `EMAIL_` setting prefix](https://docs.djangoproject.com/en/5.0/ref/settings/#email-backend), [my updates in Core-CMS to use `PORTAL_`](https://github.com/TACC/Core-CMS/pull/792). This new setting is specific to both TACC and it's `staff_profile` app.

Reply:
Fixed.

Reply:
I'm just looking for you to pick one, or offer something else. And I was probably to cryptic with my use of `…`.

- `TACC_STAFF_PROFILE_EXCLUDED_POST_NOMIAL_LIST`
- `TACC_STAFF_EXCLUDED_POST_NOMIAL_LIST`
- (or suggest an alternative)